canonical/ubuntu.com
# /certified/why-certified refresh WD-738

## Done

- Updated /certified/why-certified to match the [design](https://www.figma.com/file/PhpfN0q9qqLLHgodMYm3lV/Why-certified%3F?node-id=160%3A6625&t=kzFgx4d0kwDwC8mQ-0)

## QA

- Visit /certified/why-certified
- See that it matches the design and the copy doc

## Issue / Card

Fixes [WD-767](https://warthogs.atlassian.net/jira/software/c/projects/WD/boards/932?modal=detail&selectedIssue=WD-767)
